docs/docs/deployment.md

```diff
@@ -47,28 +38,6 @@ or via `just`:
 just deploy
 ```
 
-## Backups
```

Comment:
These are managed through Digital Ocean as part of their managed service. We do need to learn up our `justfile` to create a better pg_dump + pg_restore recipe, but production is isolated by design via DO.